### ISSUE TITLE
running pytorch model on android phone

### ISSUE THREAD

**Issue by tdas714** [pytorch/android-demo-app#41]
I've traced pytorch model.when I'm running the model on android, I'm having this error on my android studio lgocat:
Can anyone help me with this?/ what does that mean????

 Caused by: java.lang.RuntimeException: NNPACK SpatialConvolution_updateOutput failed
    The above operation failed in interpreter, with the following stack trace:
    at code/__torch__.py:364:26
        _98 = torch.tanh_(_94)
        _99 = torch.sigmoid_(_95)
        hx4 = torch.add_(torch.mul(_97, hx1), torch.mul(_96, _98), alpha=1)
        input15 = torch.mul(_99, torch.tanh(hx4))
        query = torch.dropout(input15, 0.10000000000000001, False)
        _100 = [torch.unsqueeze(_84, 1), torch.unsqueeze(_86, 1)]
        input16 = torch.cat(_100, 1)
        input17 = torch.unsqueeze(query, 1)
        processed_query = torch.matmul(input17, torch.t(weight5))
        processed_attention = torch._convolution(input16, _35, None, [1], [15], [1], False, [0], 1, False, False, True)
                              ~~~~~~~~~~~~~~~~~~ <--- HERE
        input18 = torch.transpose(processed_attention, 1, 2)
        processed_attention_weights = torch.matmul(input18, torch.t(weight8))
        _101 = torch.add(processed_query, processed_attention_weights, alpha=1)
        _102 = torch.add(_101, processed_memory, alpha=1)
        input19 = torch.tanh(_102)
        energies = torch.matmul(input19, torch.t(weight7))
        input20 = torch.squeeze(energies, -1)
        attention_weights = torch.softmax(input20, 1, None)
        _103 = torch.unsqueeze(attention_weights, 1)

**Comment by IvanKobzarev** [comment]
Hello @tdas714 
Could you please provide model / minimal py code to reproduce it?

**Comment by tdas714** [comment]
sorry, but I solved it.I'm closing this.
